templates/az-oci-adbs/main.tf

```diff
@@ -0,0 +1,46 @@
+resource "azurerm_resource_group" "resource_group" {
+  location = var.location
+  name     = var.resource_group
+}
+
+module "adbs_network" {
```

Comment:
Given this template is intended for existing vnet, this block should only provision vnet + subnet when var.virtual_network_name and var.delegated_subnet_name is null by default.